src/components/BigNumberTextV2.tsx

```diff
@@ -154,11 +154,11 @@ const BigNumberTextV2: FC<BigNumberTextV2Props> = (props) => {
                 >
                     {valueLabel}
                 </Heading>
-                {unit && value && (
+                {unitLabel !== '' && unit && value && (
```

Comment:
This fixed an issue with short-circuiting that was causing `unitLabels` with `0` value to appear in this part of the UI.